src/components/HiddenTaskListItem/HiddenTaskListItem.jsx

```diff
@@ -0,0 +1,126 @@
+const React = require('react')
+const { shell } = require('electron')
+const { connect } = require('react-redux')
+
+class HiddenTaskListItem extends React.Component {
+  onChange(event) {
+    const { storageKey } = this.props
+    const type = event.target.checked ? 'TASKS_SELECT' : 'TASKS_DESELECT'
+
+    this.props.dispatch({ type, task: { storageKey } })
+  }
+
+  openExternal(event) {
+    event.preventDefault()
+    const { url } = this.props
+    shell.openExternal(url)
+  }
+
+  iconClass() {
+    const { state, isPullRequest } = this.props
+    const iconClasses = ['octicon']
+    if (isPullRequest) {
+      iconClasses.push('octicon-git-pull-request')
+      if (state === 'open') {
+        iconClasses.push('opened')
+      } else if (state === 'closed') {
+        iconClasses.push('closed')
+      }
+    } else {
+      if (state === 'open') {
+        iconClasses.push('octicon-issue-opened')
+      } else if (state === 'closed') {
+        iconClasses.push('octicon-issue-closed')
+      }
+    }
+    return iconClasses.join(' ')
+  }
+
+  render() {
+    const { updatedAt, repository, title, repositoryOwner, user, storageKey,
+            url, state, repositoryOwnerAvatar, userAvatar } = this.props
+
+    return (
```

Comment:
I wonder if this could be made more modular. I feel like we're using nearly the same render as the regular task-list-item component.